src/doc_builder/commands/build.py

```diff
@@ -79,7 +79,7 @@ def build_command(args):
     # `version` will always start with prefix `v`
     # `version_tag` does not have to start with prefix `v` (see: https://github.com/huggingface/datasets/tags)
     version_tag = version
-    if args.version is None and "dev" not in version:
+    if args.version is None and version != default_version:
```

Comment:
No, the whole test should be `version != default_version` as `args.version` being `None` has already been handled.

Reply:
oh my bad https://github.com/huggingface/doc-builder/pull/204 😊